odoo/tools/translate.py

```diff
@@ -1166,3 +1166,36 @@ def load_language(cr, lang):
     env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
     installer = env['base.language.install'].create({'lang': lang})
     installer.lang_install()
+
+def _format_time_ago(env, time):
+    # Set Context to be used by translate _get_Lang (by introspection)
+    # DO NOT REMOVE THIS context VARIABLE
+    context = env.context
+    seconds = time.total_seconds()
+    days = round(seconds / (60 * 60 * 24))
+
+    if not days:
+        if seconds < 10:
+            return _("just now")
+        if seconds / 60 < 1:
+            return _("%s seconds ago") % round(seconds)
```

Comment:
Is it really important to be that precise? If less than 2 minutes ago, you are probably fine with "a few moments ago"